src/main/java/seedu/duke/Duke.java

```diff
@@ -28,7 +28,8 @@ public static void main(String[] args) throws DukeExceptions {
                 String output = commandInput.executeCommand(projects);
                 ui.printOutput(output);
                 isExit = commandInput.isExit();
-            } catch (NullPointerException | StringIndexOutOfBoundsException | ArrayIndexOutOfBoundsException e) {
+            } catch (NullPointerException | StringIndexOutOfBoundsException | ArrayIndexOutOfBoundsException
+                    | DukeExceptions e) {
```

Comment:
Can consider changing the exception name to something else other than Duke, unless we are still naming our program Duke. (small issue)

Reply:
yeah i'll do that once we finalize the project name, just kept it as duke exceptions because that's the intial name we used to create the class